src/content/zh-CN/2019/accessibility.md

```diff
@@ -0,0 +1,309 @@
+---
+part\_number: II
+chapter\_number: 9
+title: 无障碍可访问性
+description: 2019 Web Almanac 网络年鉴的无障碍易访问性章节，涵盖易阅读、媒体、易于导航和与辅助技术的兼容性。
+authors: [nektarios-paisios, obto, kleinab]
+reviewers: [ljme]
+translators: [chengxicn]
+discuss: 1764
+results: https://docs.google.com/spreadsheets/d/16JGy-ehf4taU0w4ABiKjsHGEXNDXxOlb\_\_idY8ifUtQ/
+queries: 09\_Accessibility
+published: 2019-11-11T00:00:00.000Z
+last\_updated: 2020-05-27T00:00:00.000Z
+---
+
+## 介绍
+
+互联网的无障碍可访问性对一个包容和公平的社会至关重要。随着我们的社会生活和工作生活越来越多地转移到网络世界，残疾人能够无障碍地参与所有在线互动就变得更加重要了。正如建筑设计师可以创建或省略无障碍功能，例如轮椅坡道，web开发人员可以帮助或阻碍用户依赖的辅助技术。
```

Comment:
Should we do a search to replace all instances of `,` with `、`?

Note line 6 needs to use a "," comma.